src/mlpack/tests/augmented_rnns_tasks_test.cpp

```diff
@@ -54,8 +54,10 @@ class HardCodedCopyModel
       size_t& addVar = (input.at(i, 0) == 0) ? zeroCnt : oneCnt;
       ++addVar;
     }
-    assert(oneCnt % zeroCnt == 0);
-    nRepeats = oneCnt / zeroCnt;
+
+      assert(zeroCnt != 0) 
```

Comment:
@gaurav1086 you are missing a semicolon here.Might be the reason for the error. 
Thanks.